Bugsnag/Payload/BugsnagError.m

```diff
@@ -132,6 +128,33 @@ + (BugsnagError *)errorFromJson:(NSDictionary *)json {
     return error;
 }
 
+- (void)updateWithCrashInfoMessage:(NSString *)crashInfoMessage {
+    @try {
+        // Messages that match this pattern should override the errorClass (and errorMessage if there is enough information.)
+        NSString *pattern = @"^(Assertion failed|Fatal error|Precondition failed): ((.+): )?file .+, line \\d+\n$";
+        NSRegularExpression *regex = [[NSRegularExpression alloc] initWithPattern:pattern options:0 error:nil];
```

Comment:
This should be [case insensitive](https://github.com/bugsnag/bugsnag-cocoa/commit/add71b3d91b672d5cb8db540f189f1529bdcc817) to be compatible with every version of swift.

```suggestion
        NSRegularExpression *regex = [[NSRegularExpression alloc] initWithPattern:pattern options:NSRegularExpressionCaseInsensitive error:nil];
```

https://github.com/apple/swift/commit/d03a575279cf5c523779ef68f8d7903f09ba901e